be/src/runtime/routine_load/data_consumer.cpp

```diff
@@ -239,9 +241,11 @@ Status KafkaDataConsumer::group_consume(TimedBlockingQueue<RdKafka::Message*>* q
         case RdKafka::ERR_OFFSET_OUT_OF_RANGE: {
             done = true;
             std::stringstream ss;
-            ss << msg->errstr() << ", partition " << msg->partition() << " offset " << msg->offset() << " has no data";
+            ss << msg->errstr() << ", this is no data in partition: " << msg->partition()
+               << " at offset: " << msg->offset();
             LOG(WARNING) << "kafka consume failed: " << _id << ", msg: " << ss.str();
-            st = Status::InternalError(ss.str());
+            st = Status::InternalError(fmt::format(
+                    "{}. you can modify kafka_offsets by alter routine load, then resume the job", ss.str()));
```

Comment:
How about giving a document link for how to alter routine load?

Reply:
done